terraform-provider-intro/overview.md

```diff
@@ -0,0 +1,52 @@
+In this scenario, you use a Terraform provider to interact with a fictional coffee-shop application, HashiCups, by completing the following actions:
+
+1. Initialize Terraform Workspace
+1. Create a HashiCups order
+1. Update a HashiCups order
+1. Read coffee ingredients (data source)
+1. Delete a HashiCups order
+
+This scenario includes a pre-installed Terraform 0.13, a pre-installed Terraform HashiCups provider and an instance of the HashiCups API running.
+
+### Verify HashiCups API is running
+
+Once you see the HashiCups logs running in your terminal, verify that HashiCups is running.
+
+Click on the following command to send a request to HashiCup's health check endpoint in another terminal.
+
+`curl localhost:19090/health`{{execute T2}} 
+
+### Create HashiCups user
+
+Create a user on HashiCups named `education` with the password `test123`.
+
+`curl -X POST localhost:19090/signup -d '{"username":"education", "password":"test123"}'`{{execute T2}}
+
+This will return the userID, username, and a JWT token. Your JWT authorization token will be used later to retrieve your orders.
```

Comment:
```suggestion
This will return the userID, username, and a JSON Web Token (JWT) authorization token. You will use the JWT authorization token later to retrieve your orders.
```
"JWT token" is sort of like "ATM machine", but I think it's ok?

Reply:
Also passive voice was used. :)